### PR TITLE
PLA 1610 cloud cmd

### PR DESCRIPTION
**Pull request by coolguru**

### CODE REVIEW

File: cmd/util/flags.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+	}
+
+	return nil
+}
+
+func checkGitRepoUrl(gitRepoUrl string) error {
+	if gitRepoUrl == "" {
+		return fmt.Errorf("A SSH git repo url is required: -g")
+	} else if !strings.Contains(gitRepoUrl, "git@") {
+		return fmt.Errorf("Use a SSH git repo url for example : [-g git@github.com:CloudCoreo/audit-aws.git]")
```
Comment (pauldallen) [inline cmd/util/flags.go]: externalize error messages

File: cmd/util/flags.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	if flag == "" {
+		return fmt.Errorf(error)
+	}
+
+	return nil
+}
+
+func checkGitRepoUrl(gitRepoUrl string) error {
+	if gitRepoUrl == "" {
+		return fmt.Errorf("A SSH git repo url is required: -g")
```
Comment (pauldallen) [inline cmd/util/flags.go]: externalize error messages

File: cmd/util/flags.go (hunk 1 above)
Comment (coolguru) [inline cmd/util/flags.go]: resolved

File: cmd/util/flags.go (hunk 2 above)
Comment (coolguru) [inline cmd/util/flags.go]: resolved

**Pull request merged**